payloads/library/GitBunnyGit/payload.txt

```diff
@@ -18,19 +18,19 @@ LED R G B
 ATTACKMODE RNDIS_ETHERNET 
 #ATTACKMODE ECM_ETHERNET
 
-source bunny_helpers.sh
-
 # Set your desired repo url and branch if you're not looking to update from mainline
 git_repo="https://github.com/hak5/bashbunny-payloads.git"
 git_branch="master"
 payloads_dir="/root/udisk"
-log_file="/root/udisk/payloads/$SWITCH_POSITION/git.log"
+log_file="/root/udisk/git.log"
+
+echo " ---------  Git Bunny Git [started] -----------" >> $log_file
```

Comment:
I personally want the log to grow. It can always be removed, and I bet they'll fix the date problem at some point; It'd be nice to have a timestamp in the log as well.